azure-pipelines/build-release.yml

```diff
@@ -84,6 +84,7 @@ stages:
 
           # Static analysis is different for Python versions so should be run in each environment.
           - script: |
+              flake8 --version
```

Comment:
Adding `flake8 --version` doesn't have much to do with removing assert-news, so it should probably go in its own commit, if we want to add it.

Reply:
Good catch. I just added it for debugging. Removed now.